test.rb

```diff
@@ -0,0 +1,19 @@
+a = 1
+  class AAA
+    def initialize
+      @a
+    end
+  
+    def a
+      @a
+    end
+  
+      def foo
+        bar
+      end
+  end
+  
+  a = ['x','y','z']
+  puts a
```

Comment:
__[rubocop]__
* Inconsistent indentation detected.
* Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)

<!-- Comment by [SideCI](https://sideci.com) -->